database/scrapers/academic_calendar_scraper.ts

```diff
@@ -0,0 +1,89 @@
+import { DateTime } from "luxon";
+
+import { BaseScraperModule, TaskHandle } from "#commands/db_scrape";
+import { Weekday } from "#enums/weekday";
+import AcademicCalendar from "#models/academic_calendar";
+
+interface AcademicCalendarData {
+  data: {
+    id: number;
+    user_updated: string;
+    date_updated: DateTime;
+    semesterStartDate: DateTime;
+    examSessionStartDate: DateTime;
+    isFirstWeekEven: boolean;
+    examSessionLastDay: DateTime;
+  };
+}
+
+interface WeekExceptions {
+  data: {
+    id: number;
+    day: DateTime;
+    changedWeekday: string;
+    changedDayIsEven: boolean;
+  }[];
+}
+
+export default class AcademicCalendarScraper extends BaseScraperModule {
+  static name = "academic calendar scraper";
+  static description = "Import data about academic calendars and day swaps";
+
+  async run(task: TaskHandle): Promise<void> {
+    task.update("Fetching academic calendar and day swaps data");
```

Comment:
You can set the initial task status message by setting the `taskTitle` static prop.
Since this script is just a single step, I don't think you need to output any additional status updates.